<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<!-- Set the presentation title -->
		<title>Data ethics in practice</title>

		<!-- Add a favicon-->
		<link type="image/x-icon" rel="shortcut icon" href="logos/DataScienceCampus_logo_square.png" />

		<link rel="stylesheet" href="../dist/reset.css">
		<link rel="stylesheet" href="../dist/reveal.css">
		<link rel="stylesheet" href="../dist/theme/black.css" id="theme">

		<!-- Theme used for syntax highlighted code -->
		<link rel="stylesheet" href="../plugin/highlight/monokai.css" id="highlight-theme">

		<!-- Local overrides of styles -->
		<link rel="stylesheet" href="reveal-override.css"/>
	</head>
	<body>
		<div class="reveal">
			<div class="slides">
				
				<!-- Slide 1: title -->
				<section>
					
					<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">OFFICIAL</p>

					<br><br><br>
					
					<h3>Data ethics in practice</h3>
					<p>
						<small>Created by <a href="https://josephcrispell.github.io/">Joseph Crispell</a> using the <a href="https://revealjs.com/">reveal.js</a> framework</small>
					</p>

					<br><br><br>

					<img align="left" src="logos/DataScienceCampus_logo.png" width="20%">&nbsp;
					<img src="logos/ONS_logo.webp" width="30%">&nbsp;
					<img align="right" src="logos/FCDO-white_logo.png" width="20%">

					<aside class="notes">
						Thanks to the organisers for inviting Olivia and I.<br><br>

						Data scientist within the Foreign, Commonwealth, & Development Office (FCDO)<br><br>

						Talk about different ethical considerations in projects I've worked on to illustrate data ethics in practice.
					</aside>
				</section>

				<!-- Section 2 -->
				<section>

					<!-- Slide 2.1: my career -->
					<section>
						<h2>My Career</h2>
						<ul>
							<li>Zoology</li>
							<li>Pathogen transmission</li>
							<li>Pathogen genomics</li>
							<li>Foreign, Commonwealth and Development Office</li>
						</ul>

						<aside class="notes">
							Started out studying Zoology at the University of Glasgow<br><br>

							Went on to study pathogen transmission during my PhD<br><br>

							Focused in on the use of genomics to track transmission.<br><br>

							Joined FCDO back in February to use data science to help AID operations around the world.
						</aside>
					</section>

					<!-- Slide 2.2: my website -->
					<section data-background-iframe="https://josephcrispell.github.io/" data-background-interactive data-preload></section>
				</section>

				<!-- Slide 3: what is data ethics -->
				<section>
					<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Words drawn from: <a href="http://doi.org/10.5281/zenodo.3233986">Data Ethics Framework</a></p>
					<a href="https://www.gov.uk/government/publications/data-ethics-framework/data-ethics-framework"><img src="images/dataEthics_wordcloud.png" alt="snp array" style="height: 80%; width: auto"></a>

					<aside class="notes">
						Olivia has a done a great job of defining and exploring data ethics.<br><br>

						What I wanted to do here was reflect on what I consider to be data ethics.<br><br>

						For me, it's the exploring and understanding the good and bad of working with data.<br><br>
						
						What should we be thinking about as we store, process and report our data?
					</aside>
				</section>

				<!-- Section 4 -->
				<section>

					<!-- Slide 4.1: cancer panels -->
					<section>
						<h2>Improving Cancer mutation panels</h2>
						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">ACCURACY</p>

						<aside class="notes">
							One of the first projects I worked on with with a bioinformatics company in Germany as an intern.<br><br>

							I worked in a team that was analysing data for building cancer panels - identifying mutations in our genomes associated with cancers<br><br>

							What was critical in this work was accuracy - can we accurately identify mutations associated with particular cancers?
						</aside>
					</section>

					<!-- Slide 4.2: SNP array -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Image credit: <a href="https://www.illumina.com/techniques/popular-applications/genotyping.html">Illumina</a></p>
						<img src="https://www.illumina.com/content/dam/illumina-marketing/images/techniques/infinium-xt-web-graphic.jpg" alt="snp array" style="height: 80%; width: auto">

						<aside class="notes">
							To identify mutations, the team compared sequencing data from people with and without cancers.<br><br>

							The aim was to develop a panel of mutations that could be used to identify individuals at risk of particular cancers.
						</aside>
					</section>

					<!-- Slide 4.3: false positives -->
					<section>
						<table>
							<thead>
								<tr>
									<th>Result \ Truth</th>
									<th>Positive</th>
									<th>Negative</th>
								</tr>
							</thead>
							<tbody>
								<tr>
									<td>Positive</td>
									<td>True positive</td>
									<td style="color:red; border-color: white;">False positive</td>
								</tr>
								<tr>
									<td>Negative</td>
									<td>False negative</td>
									<td>True negative</td>
								</tr>
							</tbody>
						</table>

						<aside class="notes">
							A key consideration here was sequencing quality. Sequencing machines can make mistakes.<br><br>

							Need to distinguish real mutations from sequencing errors.
						</aside>
					</section>

					<!-- Slide 4.4: VCF file -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Variant Call Format (VCF) file <a href="https://en.wikipedia.org/wiki/Variant_Call_Format#:~:text=The%20Variant%20Call%20Format%20(VCF,as%20the%201000%20Genomes%20Project.">specification</a></p>
						
						<pre><code class="plaintext" data-trim data-line-numbers="19-24">
							##fileformat=VCFv4.3
							##fileDate=20090805
							##source=myImputationProgramV3.1
							##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta
							##contig=&lt;ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo sapiens",taxonomy=x&gt;
							##phasing=partial
							##INFO=&lt;ID=NS,Number=1,Type=Integer,Description="Number of Samples With Data"&gt;
							##INFO=&lt;ID=DP,Number=1,Type=Integer,Description="Total Depth"&gt;
							##INFO=&lt;ID=AF,Number=A,Type=Float,Description="Allele Frequency"&gt;
							##INFO=&lt;ID=AA,Number=1,Type=String,Description="Ancestral Allele"&gt;
							##INFO=&lt;ID=DB,Number=0,Type=Flag,Description="dbSNP membership, build 129"&gt;
							##INFO=&lt;ID=H2,Number=0,Type=Flag,Description="HapMap2 membership"&gt;
							##FILTER=&lt;ID=q10,Description="Quality below 10"&gt;
							##FILTER=&lt;ID=s50,Description="Less than 50% of samples have data"&gt;
							##FORMAT=&lt;ID=GT,Number=1,Type=String,Description="Genotype"&gt;
							##FORMAT=&lt;ID=GQ,Number=1,Type=Integer,Description="Genotype Quality"&gt;
							##FORMAT=&lt;ID=DP,Number=1,Type=Integer,Description="Read Depth"&gt;
							##FORMAT=&lt;ID=HQ,Number=2,Type=Integer,Description="Haplotype Quality"&gt;
							#CHROM POS      ID         REF   ALT    QUAL  FILTER   INFO                             FORMAT       NA00001         NA00002          NA00003
							20     14370    rs6054257  G     A      29    PASS    NS=3;DP=14;AF=0.5;DB;H2           GT:GQ:DP:HQ  0|0:48:1:51,51  1|0:48:8:51,51   1/1:43:5:.,.
							20     17330    .          T     A      3     q10     NS=3;DP=11;AF=0.017               GT:GQ:DP:HQ  0|0:49:3:58,50  0|1:3:5:65,3     0/0:41:3
							20     1110696  rs6040355  A     G,T    67    PASS    NS=2;DP=10;AF=0.333,0.667;AA=T;DB GT:GQ:DP:HQ  1|2:21:6:23,27  2|1:2:0:18,2     2/2:35:4
							20     1230237  .          T     .      47    PASS    NS=3;DP=13;AA=T                   GT:GQ:DP:HQ  0|0:54:7:56,60  0|0:48:4:51,51   0/0:61:2
							20     1234567  microsat1  GTC   G,GTCT 50    PASS    NS=3;DP=9;AA=G                    GT:GQ:DP     0/1:35:4        0/2:17:2         1/1:40:3
						</code></pre>

						<aside class="notes">
							Sequencing machines thankfully give us a lot of quality information.<br><br>
							
							With these data we can identify common signatures of sequencing errors.
						</aside>
					</section>
				</section>

				<!-- Section 5 -->
				<section>
					<!-- Slide 5.1: TB in New Zealand -->
					<section>
						<h2>Cattle, possums, and bTB in New Zealand</h2>
						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">SAMPLING</p>

						<aside class="notes">
							Another project investigated bovine tuberculosis in cattle.<br><br>

							A key consideration in this project, although only clear at the end was sampling design.
						</aside>
					</section>

					<!-- Slide 5.2: New Zealand cattle -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Image credit: <a href="https://commons.wikimedia.org/wiki/File:Cattle_stop_sign_-_New_Zealand.jpg">Wikimedia</a></p>
						<img src="https://upload.wikimedia.org/wikipedia/commons/d/d8/Cattle_stop_sign_-_New_Zealand.jpg" alt="cattle" style="width: 75%">

						<aside class="notes">
							Bovine tuberculosis is a problem in cattle populations worldwide.<br><br>

							It is a disease of the lungs. In many countries controlling bovine tuberculosis means culling thousands of cattle each year.<br><br>

							The bacteria that causes bovine tuberculosis infects cattle but also many other animals.<br><br>
						</aside>
					</section>

					<!-- Slide 5.3: New Zealand possums -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Image credit: <a href="https://commons.wikimedia.org/wiki/File:Brushtail_Possum_-_AndrewMercer_IMG40144.jpg">Wikimedia</a></p>
						<img src="https://upload.wikimedia.org/wikipedia/commons/a/a7/Brushtail_Possum_-_AndrewMercer_IMG40135.jpg" alt="possum" style="width: 75%">

						<aside class="notes">
							I joined a team in New Zealand wanting to understand what role possum populations were playing.<br><br>
							
							The brushtail possum in New Zealand is a pest species introduced because of its fur.<br><br>

							There are now 30 million possums in New Zealand and they are often infected with the bacteria causing bovine tuberculosis.
						</aside>
					</section>

					<!-- Slide 5.4: sampling locations -->
					<section>
						<img src="images/newzealand_sampling.png" alt="cattle" style="width: 70%">

						<aside class="notes">
							A team in New Zealand had sampled hundreds of infected cattle and wildlife from across New Zealand.<br><br>

							Unfortunately, to address the question of transmission we needed samples from cattle and wildlife living in the same area at the same time.<br><br>

							So we couldn't quantify the direction of transmission but we did find that possums and cattle were often infected with the same strain and strong evidence for transmission to and from cattle.
						</aside>
					</section>

					<!-- Slide 5.5: NZ article -->
					<section data-background-iframe="articles/NZ_article.pdf" data-background-interactive data-preload></section>
				</section>

				<!-- Section 6 -->
				<section>
					<!-- Slide 6.1: woodchester -->
					<section>
						<h2>Cattle, badgers, and bTB in Woodchester Park</h2>
						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">REPRESENTATIVENESS</p>

						<aside class="notes">
							Another opportunity to study the transmission of the bacteria causing bovine tuberculosis came up in England.<br><br>

							Here, it isn't possums that are implicated, it's badgers.<br><br>

							Also, in England it is a much bigger problem. Thousands of herds are infected!<br><br>

							What was critical with there analyses was representativeness, understand where our data came from and how representative our results were of other areas.
						</aside>
					</section>
					
					<!-- Slide 6.2: news articles -->
					<section>
						<div class="r-stack">
							<img src="images/BTB_bbc.png" width="550" height="auto">
							<img class="fragment" src="images/BTB_guardian.png"  width="650" height="auto">
							<img class="fragment" src="images/BTB_gov.png" width="450" height="auto">
						</div>
						
						<aside class="notes">
							At least part of the problem is political!<br><br>

							Almost every week there is some controversy about the different control options in the UK.<br><br>

							There was loads of evidence that cattle and badgers were infected with similar strains of bacteria but none saying who was infecting whom?
						</aside>
					</section>

					<!-- Slide 6.3: embed google map -->
					<section data-background-iframe="https://www.google.com/maps/d/u/0/embed?mid=1QGnW6j9L3AXn0YM0rUoQc2-qQm3epdXV" data-background-interactive data-preload>
						<aside class="notes">
							A major difference to the previous work in New Zealand was the size of the sampling areas.<br><br>

							Instead of hundreds of samples from across an entire country, we had hundreds of samples from a tiny 10km squared area in the South West of England.<br><br>

							With these data we were able analyse the movements of cattle and badgers alongside the similarlity of their bacteria and found that the bacteria was being transmitted between the cattle and badgers and most frequently from badgers to cattle.<br><br>

							Importantly though, we sampled a tiny area in the South West of England. Our results while important couldn't tell us much about what was going on in the rest of the country.<br><br>

							Our sampling area was too small and similar projects in other areas would be needed.
						</aside>
					</section>

					<!-- Slide 6.4: WP article -->
					<section data-background-iframe="articles/WP_article.pdf" data-background-interactive data-preload></section>	
				</section>

				<!-- Section 7 -->
				<section>
					<!-- Slide 7.1: cote d'ivoire -->
					<section>
						<h2>Mapping HIV in C&ocircte d'Ivoire</h2>
						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">ANONYMITY</p>

						<aside class="notes">
							A project I'm currently working really highlights the importance of anonymity.<br><br>

							I've joined a team hoping to map HIV risk in adolescents in Cote d'Ivoire in West Africa.<br><br>

							Every five or so years, comprehensive national surveys are conducted to quantify HIV prevalence.<br><br>

							We want to see whether we can delve into these data to establish indicators of risk that was more readily available and more regularly updated to get more rapid signals of risk.
						</aside>
					</section>

					<!-- Slide 5.2: population density -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Data source: <a href="https://data.humdata.org/dataset/highresolutionpopulationdensitymaps-civ">Data for good</a></p>
						<img src="images/cotedivoire_popDensity.png" alt="density" style="width: 70%">

						<aside class="notes">
							However there are two important aspects we are considering while we are planning this project:<br><br>
							1. How can we protect those vulnerable groups we aim to identify? From stigma for example.<br>
							2. With highly sensitive data like HIV test results and socio-economic status, how can we protect our data subjects identities?
						</aside>
					</section>

					<!-- Slide 5.3: displacement -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Image source: <a href="https://data.humdata.org/dataset/highresolutionpopulationdensitymaps-civ">Demographic Household Surveys</a></p>
						<img src="https://www.researchgate.net/profile/Clara_Burgert/publication/291343742/figure/fig1/AS:320352325521409@1453389354194/DHS-household-survey-displacement-process.png" alt="displacement" style="width: 70%">

						<aside class="notes">
							Methods such as random displacement can be used to protect individuals by masking their locations.<br><br>

							Any method weighs the importance of anonmynity against the utility of the infromation in the data.
						</aside>
					</section>
				</section>

				<!-- Section 8 -->
				<section>
					<!-- Slide 8.1: RAP -->
					<section>
						<h2>Reproducible data science</h2>

						<aside class="notes">
							Lastly, I want to touch upon a really important aspect of my work as a data scientist.<br><br>

							As a team we are teaching about what the use and importance of reproducible data science, which is essential to ensure we are considering all aspects of data ethics.
						</aside>
					</section>

					<!-- Slide 8.2: transparency -->
					<section>
						<pre><code class="json" data-trim data-line-numbers="19-24">
							{
								"title": "Cote d'Ivoire: High Resolution Population Density Maps + Demographic Estimates. Population and Youth (15-24)",
								"source": "Facebook",
								"url": "https://data.humdata.org/dataset/highresolutionpopulationdensitymaps-civ",
								"type": "tif/csv",
								"date-of-dataset": "01-07-2019",
								"date-accessed": "23-03-2020"
							  }
						</code></pre>

						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">BE TRANSPARENT</p>

						<aside class="notes">
							We must be transparent in our project, clearly documenting our data, our coding, our analyses, results and our intepretation.
						</aside>
					</section>

					<!-- Slide 8.3: open -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Image credit: <a href="http://doi.org/10.5281/zenodo.3233986">The Turing Way</a></p>
						<img src="images/Community.png" alt="open" style="width: 75%; background-color:white;">

						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">BE OPEN</p>

						<aside class="notes">
							We must be open. Not working in silos, but publicising our projects, joining the open-source community of sharing with and learning from others.
						</aside>
					</section>

					<!-- Slide 8.4: reproducible -->
					<section>
						<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">Image credit: <a href="http://doi.org/10.5281/zenodo.3233986">The Turing Way</a></p>
						<img src="images/ReproducibleDefinitionGrid.jpg" alt="reproducible" style="width: 75%">

						<p style="position: fixed; bottom: 0; width:100%; text-align: right; font-style: italic !important">BE MORE THAN REPRODUCIBLE</p>

						<aside class="notes">
							It isn't enough to be reproducible, we must build tools that others can easily understand, use, and adapt.
						</aside>
					</section>
				</section>

				<!-- Slide 9: resources -->
				<section>
					<h2>Resources</h2>
					<ul>
						<li>Humanitarian Data Science and Ethics Group project planning decision <a href="https://5f2cd2ba-741c-4b29-ae47-00a8291b1d3c.filesusr.com/ugd/d1cf5c_42aaabed14e04ed786c7941df0fe4484.pdf">tree</a></li>
						<li>Open Data Institute data ethics <a href="https://theodi.org/wp-content/uploads/2019/07/ODI-Data-Ethics-Canvas-2019-05.pdf">canvas</a></li>
						<li>Teaching <a href="https://github.com/datasciencecampus/gov-uk-rap-materials"> materials</a> for reproducible data science</li>
						<li>The RAP <a href="https://ukgovdatascience.github.io/rap-website/companion-index.html">companion</a> - a book about Reproducible Analytical Pipelines</li>
						<li>UK government Data Ethics <a href="https://www.gov.uk/government/publications/data-ethics-framework/data-ethics-framework">Framework</a></li>
						<li>A <a href="https://www.butterfly.com.au/blog/website-development/clean-high-quality-code-a-guide-on-how-to-become-a-better-programmer">guide</a> to how to become a better programmer</li>
					</ul>

					<aside class="notes">
						I'll finish by sharing a few resources. I've barely scratched the surface of data ethics and hopefully some of these will highlight the many other important aspects that I've missed.
					</aside>
				</section>

				<!-- Slide 10: END -->
				<section>

					<p style="position: fixed; top: 0; width:100%; text-align: left; font-size:15px">OFFICIAL</p>

					<br><br><br>
					
					<h3>THANKS FOR LISTENING!! 👂</h3>

					<br><br><br><br>

					<img align="left" src="logos/DataScienceCampus_logo.png" width="20%">&nbsp;
					<img src="logos/ONS_logo.webp" width="30%">&nbsp;
					<img align="right" src="logos/FCDO-white_logo.png" width="20%">
				</section>
			</div>
		</div>

		<script src="../dist/reveal.js"></script>
		<script src="../plugin/notes/notes.js"></script>
		<script src="../plugin/markdown/markdown.js"></script>
		<script src="../plugin/highlight/highlight.js"></script>
		<script>
			// More info about initialization & config:
			// - https://revealjs.com/initialization/
			// - https://revealjs.com/config/
			Reveal.initialize({
				hash: true,

				// Learn about plugins: https://revealjs.com/plugins/
				plugins: [ RevealMarkdown, RevealHighlight, RevealNotes ]
			});
		</script>
	</body>
</html>